examples/zephyr/fw_update/src/fw_update.c

```diff
@@ -328,7 +263,7 @@ static enum golioth_status fw_write_block_cb(const struct golioth_ota_component
 
     ctx->retries = 0;
     ctx->bytes_downloaded += block_buffer_len;
-    golioth_sys_sha256_update(ctx->sha, block_buffer, block_buffer_len);
+    mbedtls_sha256_update(&ctx->sha, block_buffer, block_buffer_len);
```

Comment:
Can we use multi-part PSA hash operation (e.g. [`psa_hash_update()`](https://arm-software.github.io/psa-api/crypto/1.1/api/ops/hashes.html#c.psa_hash_update)) instead of mbedtls APIs in the fw update example?